template/pkg/service.sls

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
```

Comment:
`pkg` subdir should not have a service management state file. See that this service file should be moved to `template/service/init.sls`, as per [@noelmcloughlin ' s proposed structure](https://github.com/saltstack-formulas/template-formula/pull/48#pullrequestreview-207182085)

Reply:
As opened for discussion in the [comment above](https://github.com/saltstack-formulas/template-formula/pull/48#discussion_r259797184).